store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -2389,7 +2444,9 @@ private static void fillVarsWithDomainAttrs(DomainAttrItem item)
                 vip = InetAddress.getByName(item.virtualIPAddress);
             }
         } catch (UnknownHostException e) {
-            throw new ProxyConfException("virtual host name \"" + item.virtualHostname + "\" is not resolvable", e);
+            mLog.warn("virtual host name \"" + item.virtualHostname + "\" is not resolvable");
+            // TODO: Allow strict enforcement as an argument
+            //throw new ProxyConfException("virtual host name \"" + item.virtualHostname + "\" is not resolvable", e);
```

Comment:
Does this TODO need to be addressed before committing this change?  Seems likely as no longer throwing an exception.

Reply:
I am unsure if we want to enforce this any longer.  The issue is if addresses are placed in say /etc/hosts they will not necessarily resolve on the proxy server but should still be handled by the 'Host' header value.
If the value fails to resolve and throws the exception then it prevents successful completion of the configuration generation process.

Reply:
I mean the client may expect them to be handled properly on the server even without dns resolution. 
In reality it seemed the DNS lookup was used merely to specify the 'listen' address for the server block.
Loosening this up doesn't seem to cause great issues because if you want to make it work the old way you simply make some DNS entries for the hostnames.

Reply:
@plobbes Didn't want me to add yet another LDAP attribute to control this resolution enforcement.

Reply:
Updated to add a `-f` `--force-dns-resolution` command-line argument to enable the old behavior.